tests/services/test_encryption_service.py

```
@@ -41,80 +39,80 @@ def test_encrypt_decrypt_file(self, db_session, tmp_path):
+
         file_encryption_service.encrypt_file(db_session, input_file, encrypted_file)
-        
+
         # Corrupt one byte of ciphertext (after salt[16] and nonce[12])
         data = bytearray(encrypted_file.read_bytes())
-        data[30] ^= 0xFF 
+        data[30] ^= 0xFF
         encrypted_file.write_bytes(data)
-        
+
         with pytest.raises(Exception):  # cryptography.exceptions.InvalidTag
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)